Doppler.ContactPolicies.Api/Controllers/ContactPolicies.cs

```diff
@@ -36,9 +36,8 @@ public async Task<IActionResult> GetContactPoliciesSettings(string accountName)
         public async Task<IActionResult> UpdateContactPoliciesSettings(string accountName,
                     [FromBody] ContactPoliciesSettingsDto contactPoliciesSettings)
         {
-            // TODO: replace this validation to get UserId by accountName
-            var currentContactPoliciesSetting = await _contactPoliciesService.GetContactPoliciesSettingsAsync(accountName);
-            if (currentContactPoliciesSetting == null)
+            var userFounded = await _contactPoliciesService.GetIdUserByAccountName(accountName);
+            if (userFounded == null)
                 return NotFound($"Account {accountName} does not exist.");
 
             await _contactPoliciesService.UpdateContactPoliciesSettingsAsync(accountName, contactPoliciesSettings);
```

Comment:
[The same here](https://github.com/FromDoppler/doppler-contact-policies-api/pull/23/files#r662222364):

```suggestion
            await _contactPoliciesService.UpdateContactPoliciesSettingsAsync(idUser, contactPoliciesSettings);
```

Reply:
It will be changed in another PR. Marking as resolved...